src/panels/MinimalPanel.vue

```diff
@@ -10,42 +10,51 @@
       </button>
     </div>
     <div v-show="!localMinimized" class="card card-flex" @mouseover="onHeaderHover = true" @mouseleave="onHeaderHover = false">
-      <div :class="['header-wrapper', { 'header-wrapper-bottom': localExpanded, 'header-toggle': isExpandableCard }]"
+      <div :class="['header-wrapper', { 'header-wrapper-bottom': isHeaderAtBottom, 'header-toggle': isExpandableCard }]"
            @click.prevent.stop="isExpandableCard && toggle()">
-        <span v-show="!localExpanded" ref="headerWrapper"
-              :class="['card-title', 'card-title-transparent', { 'card-title-opaque': onHeaderHover, 'ellipses': !hasHeaderBool }]">
-          <slot name="header">
-            <span class="card-title-inline"><slot name="_header"></slot></span>
-            <span v-show="showDownSwitch" aria-hidden="true"
-                  class="minimal-button glyphicon glyphicon-menu-down minimal-menu-down"></span>
-          </slot>
-        </span>
-        <div :class="['button-wrapper', { 'button-wrapper-expanded': localExpanded, 'button-wrapper-visible': onHeaderHover }]">
+        <transition name="title">
```

Comment:
transition for making the opacity of the minimal panel slowly fade back when collapsed